tools/java/org/unicode/cldr/util/CLDRConfig.java

```diff
@@ -794,4 +794,23 @@ public static VersionInfo getVersion(int... versionInput) {
         return VersionInfo.getInstance(version[0], version[1], version[2],
             version[3]);
     }
+
+    /**
+     * Get a config value as an integer.
+     * @param key key name
+     * @param defaultValue default value
+     * @return the value, or defaultValue if missing or unparseable.
+     * @see CLDRConfig#get(Object)
+     */
+    public int get(String key, int defaultValue) {
```

Comment:
The name of this is very obscure. It appears that this calls get(key) which calls getProperty() — which does have a meaningful name. The get() methods should be renamed to getProperty()

Reply:
So this is actually inherited from the fact that it extends `Properties` which extends `Hashtable`. 
But I will change this to `getProperty() which is clearer.

Reply:
It's worse than that, I actually already added a `getProperty(String, Integer)` some (years) ago.  Deleted the duplicate implementation and changing code to use that...

Reply:
Done